.github/workflows/drivers.yml

```diff
@@ -735,7 +735,7 @@ jobs:
       MB_POSTGRES_SSL_TEST_SSL_ROOT_CERT_PATH: 'test-resources/certificates/us-east-2-bundle.pem'
     services:
       postgres:
-        image: circleci/postgres:latest
+        image: postgres:latest
```

Comment:
probably a question for @metabase/core-backend-qp , but do we want to use `:latest` I would suspect we should pin to a known-supported tag and re-evaluate periodically?

Reply:
I've updated mysql to the `:latest` version In the PR https://github.com/metabase/metabase/pull/36025. As there were no objections from the rest of the team, I believe that approach is ok.